README.md

```
@@ -1 +1,40 @@
+```
+
+スクリプト経由で Windows ビルドする場合:
+```bash
+scripts/build-client-core-windows.sh
+```
+
+### 4) Godot 側の設定
+
+`client/godot/client_core.gdextension` は以下の DLL を参照します。
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)